stdlib/ballerina-builtin/src/main/ballerina/ballerina/jwt/issuer.bal

```diff
@@ -31,15 +31,16 @@ public struct JWTIssuerConfig {
 @Param {value:"config: JWTIssuerConfig object"}
 @Return {value:"string: JWT token string"}
 @Return {value:"error: If token validation fails "}
-public function issue (Header header, Payload payload, JWTIssuerConfig config) returns (string, error) {
+public function issue (Header header, Payload payload, JWTIssuerConfig config) returns (string) | error {
```

Comment:
It would be better to re-arrange the parenthesis like `(string|error)`.

Reply:
this way is correct as well right?